toolchain/arangoproxy/cmd/configs/local.yaml

```diff
@@ -1,13 +1,4 @@
-webserver: ':8080'
-logFile: log.txt
 datasetsFile: '../internal/utils/datasets.json'
 cache: '/home/site/data/'
-repositories:
-  - name: stable
-    type: single
-    version: "3.12"
-    url: http://192.168.129.2:8529
-  - name: stable
-    type: cluster
-    version: "3.12"
-    url: http://192.168.129.3:8529
+debug: false
```

Comment:
Does setting this option to true enable more logging than before and leaving it to false log the same amount of information as before, or does false log less information than before?

Reply:
Set to false will log the same info we see now. True will add more info on top of what we have now